.scalafmt.conf

```diff
@@ -11,7 +11,7 @@ To fix this problem:
 """
 
 project.excludeFilters = [
-  test-workspace
-  tests/unit/src/test/resources
-  sbt-metals/src/sbt-test
+  "test-workspace"
+  "tests/unit/src/test/resources"
+  "sbt-metals/src/sbt-test"
```

Comment:
not related, but this change is necessary for my Scalafmt VSCode extension to work (due to config parser differences in JS)